ProteGO/Common/DataModel/GetStatus/GetStatusResponse.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+struct GetStatusResponseBeaconId: Decodable {
+    let beaconId: String
```

Comment:
Rozważałeś, żeby pole `beaconId` było typu `BeaconId`? Mamy już na to strukturę.

Reply:
👍 dodałem otestowane tworzenie BeaconId zgodne z Decodable.

Przyda nam się za chwilę przy zapisywaniu tego do Realma i potem przepisywaniu do Advertisera BLE